boards/deprecated.cmake

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
@tejlmand can you add a set of CODE-OWNERs for this file?

Reply:
done